pkg/cmd/root/root.go

```diff
@@ -25,6 +25,8 @@ import (
 )
 
 func NewCmdRoot(f *cmdutil.Factory, version, buildDate string) *cobra.Command {
+	cobra.MousetrapHelpText = ""
```

Comment:
Since this is modifying global `cobra` package state, I would advise moving it to `cmd/gh/main.go` instead of executing it as part of this function.